app/guid-node/metadata/route.ts

```diff
@@ -0,0 +1,29 @@
+import Store from '@ember-data/store';
+import Route from '@ember/routing/route';
+import RouterService from '@ember/routing/router-service';
+import { inject as service } from '@ember/service';
+import CedarMetadataTemplateModel from 'ember-osf-web/models/cedar-metadata-template';
+
+export default class MetadataDetailRoute extends Route {
+    @service store!: Store;
+    @service router!: RouterService;
+
+    async model() {
+        const guidNode = this.modelFor('guid-node');
+        const target = await guidNode.taskInstance;
+
+        const cedarMetadataRecords = await target.queryHasMany('cedarMetadataRecords', {
+            // embed: 'template',
```

Comment:
Leaving a comment here to call out that we want to check with platform folks if this endpoint supports this `embed` behavior